### ISSUE TITLE
Bug: Text in the First Name textbox flickr's 

### ISSUE THREAD

**Issue by bevanamit** [theheapdump/kite-webapp#208]
Text in the First Name textbox flickr's with "Enter your first name" and the actual text which is there saved when CANCEL is clicked.

Login
Navigate to Profile Settings
Ensure the details are added and saved
Click on CANCEL couple of times, you will see the text changing from actual saved and  "Enter your first name" for every click that is done

**Comment by kshitijbhardwaj** [comment]
Yes can't  do anything about it. On cancel server call happens and data reloads again so this will be there. ALso its an highly unlikely scenario.
Please close this issue.
